### PR TITLE
(Here we go again...) Remove unnecessary dependencies

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?

Since the recent changes made in both [vtex.render-runtime](https://github.com/vtex-apps/render-runtime/pull/473) and [vtex.builder-hub](https://github.com/vtex/builder-hub/pull/856) regarding the way `allowed` and `composition` work were released, a bunch of dependencies added to `vtex.store` became unnecessary.

This pull request removes an impressive **32 dependencies** from `vtex.store`.

#### What problem is this solving?

Every app to which `vtex.store` is dependant on has its major version locked, and with this, we should be able to set them free!

#### How should this be manually tested?

- [exitocol](https://freestoredependencies--exitocol.myvtex.com/)
- [alssports](https://freestoredependencies--alssports.myvtex.com/)
- [melimeloparis](https://freestoredependencies--melimeloparis.myvtex.com/)
- [samsungar](https://freestoredependencies--samsungar.myvtex.com/)
- [ecowaterqa](https://freestoredependencies--ecowaterqa.myvtex.com/)
- [btglobal](https://freestoredependencies--btglobal.myvtex.com/)
- [tbb](https://freestoredependencies--tbb.myvtex.com/)
- [motorolauk](https://freestoredependencies--motorolauk.myvtex.com/)
- [gc-jqu0734](https://freestoredependencies--gc-jqu0734.myvtex.com/) a.k.a Madine Jeans
- [garmin](https://freestoredependencies--garmin.myvtex.com/)
- [paguemenos](https://freestoredependencies--paguemenos.myvtex.com/)
- [powerplanet](https://freestoredependencies--powerplanet.myvtex.com/)
- [boticario](https://freestoredependencies--boticario.myvtex.com/)
- [worldwidegolf](https://freestoredependencies--worldwidegolf.myvtex.com/)
- [motorolaes](https://victormiranda--motorolaes.myvtex.com/)
- [biscoind](https://victormiranda--biscoind.myvtex.com/)
- [fila](https://victormiranda--fila.myvtex.com/)
- [samsungpe](https://victormiranda--samsungpe.myvtex.com/)
- [clara](https://victormiranda--clara.myvtex.com/)
- [spaziolusso](https://victormiranda--spaziolusso.myvtex.com/)
- [samsungco](https://victormiranda--samsungco.myvtex.com/)
- [btpt](https://victormiranda--btpt.myvtex.com/)
- [festaexpress](https://victormiranda--festaexpress.myvtex.com/)
- [brcraft](https://victormiranda--brcraft.myvtex.com/)
- [mundialfarma](https://victormiranda--mundialfarma.myvtex.com/)
- [naturalessencia](https://victormiranda--naturalessencia.myvtex.com/)
- [unimarc](https://victormiranda--unimarc.myvtex.com/)
- [abelvolks](https://victormiranda--abelvolks.myvtex.com/)
- [cosmetics2](https://victormiranda--cosmetics2.myvtex.com/)
- [lojakamikasi](https://victormiranda--lojakamikasi.myvtex.com/)
- [samsungpeepp](https://victormiranda--samsungpeepp.myvtex.com/)
- [newbalance](https://victormiranda--newbalance.myvtex.com/)
- [parquedpedro](https://victormiranda--parquedpedro.myvtex.com/)
- [amalia](https://victormiranda--amalia.myvtex.com/)
- [lily](https://victormiranda--lily.myvtex.com/)
- [carulla](https://victormiranda--carulla.myvtex.com/)
- [columbiacl](https://victormiranda--columbiacl.myvtex.com/)
- [lesters](https://victormiranda--lesters.myvtex.com/)
- [gympassbr](https://victormiranda--gympassbr.myvtex.com/)
- [samsungar](https://victormiranda--samsungar.myvtex.com/)
- [indevatotens](https://victormiranda--indevatotens.myvtex.com/)
- [maissaudeebeleza](https://victormiranda--maissaudeebeleza.myvtex.com/)
- [analoren](https://victormiranda--analoren.myvtex.com/)

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -17,52 +17,20 @@
     "vtex.store-graphql": "2.x",
     "vtex.store-header": "2.x",
     "vtex.store-footer": "2.x",
-    "vtex.store-components": "3.x",
```
Comment (lbebber) [inline manifest.json]: :satisfaction:

File: store/interfaces.json [hunk 2]
```diff
@@ -5,40 +5,13 @@
     "around": ["storeWrapper", "challenge"],
     "before": ["header.full"],
     "after": ["footer", "pixels"],
-    "allowed": [
```
Comment (lbebber) [inline store/interfaces.json]: ✨

**Comment by lbebber** [comment]
alias o que quer dizer com `[NOT WORKING PROPERLY]` na garmin?

**Comment by victorhmp** [comment]
> alias o que quer dizer com `[NOT WORKING PROPERLY]` na garmin?

Problema no order-placed, exatamente o mesmo q deu na Boticario. Já falei com o Gabriel de Curitiba, eles vão resolver isso antes de lançarmos essa nova versão.

**Comment by lbebber** [comment]
(comentei pessoalmente mas só pra deixar registrado) tenta automatizar esse teste do orderPlaced pra evitar surpresas

**Comment by victorhmp** [comment]
Teste rodado!
Encontrei potênciais problemas nas lojas:
- catcl
- umbro
- eicom

**Comment by victorhmp** [comment]
Lojas atualizadas, tudo pronto para lançar! 🎉

**Pull request merged**